packs/src/items/trinket/brass-horn-of-valhalla.json

```diff
@@ -5,7 +5,7 @@
   "img": "icons/tools/instruments/flute-simple-wood.webp",
   "system": {
     "description": {
-      "value": "<p><em>Wondrous item</em></p>\n<p>You can use an action to blow this horn. In response, warrior spirits from the Valhalla appear within 60 feet of you. They use the statistics of a <a class=\"content-link\" draggable=\"true\" data-pack=\"dnd5e.monsters\" data-id=\"kz1t6xeXVwODpYb2\"> Berserker</a>. They return to Valhalla after 1 hour or when they drop to 0 hit points. Once you use the horn, it can't be used again until 7 days have passed.</p>\n<p>Four types of Horn of Valhalla are known to exist, each made of a different metal. The horn's type determines how many berserkers answer its summons, as well as the requirement for its use. The GM chooses the horn's type or determines it randomly.</p>\n<table>\n<thead>\n<tr>\n<th>d100</th>\n<th>Horn Type</th>\n<th>Berserkers Summoned</th>\n<th>Requirement</th>\n</tr>\n</thead>\n<tbody>\n<tr>\n<td>01-40</td>\n<td>Silver</td>\n<td>2d4 + 2</td>\n<td>None</td>\n</tr>\n<tr>\n<td><strong>41-75</strong></td>\n<td><strong>Brass</strong></td>\n<td><strong>3d4 + 3</strong></td>\n<td><strong>Proficiency with all simple weapons</strong></td>\n</tr>\n<tr>\n<td>76-90</td>\n<td>Bronze</td>\n<td>4d4 + 4</td>\n<td>Proficiency with all medium armor</td>\n</tr>\n<tr>\n<td>91-00</td>\n<td>Iron</td>\n<td>5d4 + 5</td>\n<td>Proficiency with all martial weapons</td>\n</tr>\n</tbody>\n</table>\n<p>If you blow the horn without meeting its requirement, the summoned berserkers attack you. If you meet the requirement, they are friendly to you and your companions and follow your commands.</p>",
+      "value": "<p><em>Wondrous item</em></p>\n<p>You can use an action to blow this horn. In response, warrior spirits from the Valhalla appear within 60 feet of you. They use the statistics of a <a class=\"content-link\" draggable=\"true\" data-pack=\"dnd5e.monsters\" data-id=\"kz1t6xeXVwODpYb2\"> Berserker</a>. They return to Valhalla after 1 hour or when they drop to 0 hit points. Once you use the horn, it can't be used again until 7 days have passed.</p>\n<p>Four types of Horn of Valhalla are known to exist, each made of a different metal. The horn's type determines how many berserkers answer its summons, as well as the requirement for its use. The GM chooses the horn's type or determines it randomly from the @Compendium[dnd5e.tables.RwerG95cIk2uRpsQ]{Horn of Valhalla} table.</p>\n<p><strong>Brass Horn of Valhalla.</strong>This brass horn summons [[/r 3d4 + 3]] berserkers when blown and requires proficiency with all simple weapons to be used.</p>\n<p>If you blow the horn without meeting its requirement, the summoned berserkers attack you. If you meet the requirement, they are friendly to you and your companions and follow your commands.</p>",
```

Comment:
It looks like the link to Berserker in this horn isn't a proper draggable link. Might as well fix it while you are here:

```suggestion
      "value": "<p><em>Wondrous item</em></p>\n<p>You can use an action to blow this horn. In response, warrior spirits from the Valhalla appear within 60 feet of you. They use the statistics of a @Compendium[dnd5e.monsters.kz1t6xeXVwODpYb2]{Berserker}. They return to Valhalla after 1 hour or when they drop to 0 hit points. Once you use the horn, it can't be used again until 7 days have passed.</p>\n<p>Four types of Horn of Valhalla are known to exist, each made of a different metal. The horn's type determines how many berserkers answer its summons, as well as the requirement for its use. The GM chooses the horn's type or determines it randomly from the @Compendium[dnd5e.tables.RwerG95cIk2uRpsQ]{Horn of Valhalla} table.</p>\n<p><strong>Brass Horn of Valhalla.</strong>This brass horn summons [[/r 3d4 + 3]] berserkers when blown and requires proficiency with all simple weapons to be used.</p>\n<p>If you blow the horn without meeting its requirement, the summoned berserkers attack you. If you meet the requirement, they are friendly to you and your companions and follow your commands.</p>",
```